ci-operator/config/openshift/etcd/openshift-etcd-origin-4.0-etcd-v3.3.10.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/etcd
+images:
+- from: base
+  to: etcd
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+tests:
+- as: unit
```

Comment:
you'll want to add an e2e-aws one here as well

Reply:
not sure I completely understand this usage let me know if I need something more custom.